### DISCUSSION TITLE
Question about SiamMask model

### DISCUSSION THREAD

**Discussion by zw615** [STVIR/pysot#15] [General]
Hi, there!
    I'm trying to reproduce the result of SiamMask in PySOT Model Zoo -- 0.423/0.615/0.248 (EAO/A/R) @VOT2018. However, using the released model I only get 0.407/0.598/0.243 (EAO/A/R) @VOT2018. Also, the average speed is about 30 FPS, which is much lower than 56FPS.
    I have tried to set __C.MASK.MASK and __C.MASK.REFINE to True，but the result does not change. 
    I wonder if I was using the wrong command/params ? Is there anythin I should pay attention to other than the Quick Start tutorial and config.py？
    Thanks a lot!

**Comment by CXSunshine** [comment]
where can I get the code of siameseFC？Thanks~

**Comment by xyl-507** [comment]
@JWarlock 
Hi! sorry to bother you.
I found you have reproduce SiamMask on paltform PYSOT. It's difficult for me to reproduce it .so I have some question:
1. how to set the parameters in config.yaml
2. `mask_loss= none`  in this rpo, how did you set?
3. Is there other note？
can you share your code？It must be helpful ! Thanks！
